calibration:
    ros__parameters:
      correspondence_path: "/ros2_ws/src/lidar_camera_calibration/data/correspondences.txt"
      k_matrix_path: "/ros2_ws/src/lidar_camera_calibration/data/zhangs_intrinsics.txt"
      output_path: "/ros2_ws/src/lidar_camera_calibration/data/output.txt"
      camera_base_frame: "camera_bottom_screw_frame"
      camera_sensor_frame: "camera_color_optical_frame"
      lidar_base_frame: "velodyne_base_link"
      lidar_sensor_frame: "velodyne"
      use_ransac: true
      ransac_iterations: 1000
      ransac_reprojection_error: 7.5
      ransac_confidence: 0.99